web/react/stores/team_store.jsx

```diff
@@ -11,6 +11,12 @@ var BrowserStore = require('../stores/browser_store.jsx');
 
 var CHANGE_EVENT = 'change';
 
+var utils;
+function getWindowLocationOrigin() {
+    if (!utils) utils = require('../utils/utils.jsx');
```

Comment:
Why not just include ```utils = require('../utils/utils.jsx');``` with the rest of the requires? It's likely going to be needed later in other functions so you don't want to have to repeat this code everywhere